_data/navigation.yml

```diff
@@ -298,6 +298,15 @@ guides:
         url: /guides/runbooks/sharding-spinnaker/
       - title: Orca Quality of Service
         url: /guides/runbooks/orca-quality-of-service/
+  - title: Spin CLI
+    collapsed: true
+    children:
+      - title: Install and Configure Spin CLI
+        url: /guides/spin/cli
```

Comment:
Minor point on the urls---technically they should have a trailing slash (as we're looking for `/guides/spin/cli/index.html`).  The site will return a 301 redirecting `/guides/spin/cli` to `/guides/spin/cli/` but it probably makes sense to just skip that round-trip.

Reply:
Done.